DataFormats/MuonReco/src/classes.h

```diff
@@ -33,7 +33,7 @@
 #include <vector>
 #include <map>
 
-namespace DataFormats_MuonReco {
+namespace {
```

Comment:
what was wrong with this?
all DataFormats/* classes are in their "localized" namespace